exercises/run-length-encoding/src/RunLength.hs

```diff
@@ -0,0 +1,9 @@
+module RunLength (decode, encode) where
+
+import Data.Char (isDigit)
+
+decode :: String -> String
+decode = undefined
+
+encode :: String -> String
+encode = undefined
```

Comment:
If at a later date the decision is made to replace `undefined` with `error` (a decision that I'd support) the change can easily be implemented with a single find-replace.